src/main/java/org/com/itpple/spot/server/controller/UserInfoController.java

```diff
@@ -43,4 +38,26 @@ public ResponseEntity<Void> nicknameCheck(@RequestParam("nickname") String nickn
         return ResponseEntity.ok().build();
     }
 
+    //유저 조회
+    @GetMapping("/info")
```

Comment:
restful api로 작성하면 좋을 거 같아요!
```suggestion
    @GetMapping("")
```

Reply:
수정했습니다!!